wsmaster/codenvy-sql-schema/src/main/resources/codenvy-schema/5.0.0-M8/1.1__init.sql

```diff
@@ -360,3 +360,30 @@ CREATE TABLE freeresourceslimit_resource (
 ALTER TABLE freeresourceslimit_resource ADD CONSTRAINT fk_freeresourceslimit_resource_resources_id FOREIGN KEY (resources_id) REFERENCES resource (id);
 ALTER TABLE freeresourceslimit_resource ADD CONSTRAINT frresourceslimitresourcefreresourceslimitaccountid FOREIGN KEY (freeresourceslimit_accountid) REFERENCES freeresourceslimit (accountid);
 --------------------------------------------------------------------------------
+
```

Comment:
Do not modify existing `.sql` file - create a new one instead, or migration will fail on existing instance with checksum verifycation exception. See [description](https://github.com/eclipse/che/pull/3063) for understanding versioning